Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -487,6 +489,45 @@ public override void Update(float elapsedClockSeconds)
             else
                 CylPressurePSI = AutoCylPressurePSI;
 
+            // During braking wheelslide control is effected throughout the train by additional equipment on each vehicle. In the piping to each pair of brake cylinders are fitted electrically operated 
+            // dump valves. When axle rotations which are sensed electrically, differ by a predetermined speed the dump valves are operated releasing brake cylinder pressure to both axles of the affected 
+            // bogie.
+
+            // Dump valve operation will cease when differences in axle rotations arewithin specified limits or the axle accelerates faster than a specified rate. The dump valve resets whenever the wheel
+            // creep speed drops to normal. The dump valve will only operate continuously for a maximum period of seven seconds after which time it will be de-energised and the dump valve will not 
+            // re-operate until the train has stopped or the throttle operated. 
+
+            // Dump valve operation is prevented under the following conditions:-
+            // (i) When the Power Controller is open.
+
+            // (ii) When Brake Pipe Pressure has been reduced below 250 kPa (36.25psi). 
+
+            if (Car.WheelBrakeSlideProtectionFitted && Car.Train.IsPlayerDriven)
+            {
+                // WSP dump valve active
+                if ((Car.BrakeSkidWarning || Car.BrakeSkid) && CylPressurePSI > 0 && !Car.WheelBrakeSlideProtectionDumpValveLockout && ( (!Car.WheelBrakeSlideProtectionLimitDisabled && BrakeLine1PressurePSI > 36.25) || Car.WheelBrakeSlideProtectionLimitDisabled) )
+                {
+                    Car.WheelBrakeSlideProtectionActive = true;
+                    AutoCylPressurePSI -= elapsedClockSeconds * MaxReleaseRatePSIpS;
+                    CylPressurePSI = AutoCylPressurePSI;
+                    Car.WheelBrakeSlideProtectionTimerS -= elapsedClockSeconds;
+
+                    // Lockout WSP dump valve if it is open for greater then 7 seconds continuously
+                    if (Car.WheelBrakeSlideProtectionTimerS <= 0)
+                    {
+                        Car.WheelBrakeSlideProtectionDumpValveLockout = true;
+                    }
+
+                }
+                else if (!Car.WheelBrakeSlideProtectionDumpValveLockout)
+                {
+                    // WSP dump valve stops
+                    Car.WheelBrakeSlideProtectionActive = false;
+                    Car.WheelBrakeSlideProtectionTimerS = 7.0f; // Reset WSP timer if 
```

Comment:
```suggestion
                    Car.WheelBrakeSlideProtectionTimerS = 7.0f; // Reset WSP timer if 
```
Car.WheelBrakeSlideProtectionTimerS is set to 7.0 secs in 3 places.
Would a const value defined in just 1 place be tidier?

Reply:
Thanks, I have added a "constant".